critiquebrainz/frontend/external/musicbrainz_db/tests/release_group_test.py

```diff
@@ -12,9 +12,10 @@ def setUp(self):
         setup_cache()
         mb_release_group.mb_session = MagicMock()
         self.mock_db = mb_release_group.mb_session.return_value.__enter__.return_value
-        self.release_group_query = self.mock_db.query.return_value.options.return_value.\
-                options.return_value.options.return_value.options.return_value.filter.return_value.all
-        self.release_group_query_without_artists = self.mock_db.query.return_value.options.return_value.filter.return_value.all
+        self.release_group_query = self.mock_db.query.return_value.options.return_value.options.return_value.\
+            options.return_value.options.return_value.options.return_value.filter.return_value.all
+        self.release_group_query_without_artists = self.mock_db.query.return_value.\
+            options.return_value.options.return_value.filter.return_value.all
```

Comment:
I have split this to make it clearer. The test function `test_get_by_id` fetches release groups with the include `artists` and the test function `test_fetch_multiple_release_groups` fetches release groups with no includes. Therefore, I've mocked two different queries, one with artists and other without artists.